src/i18n/ja/nav.ts

```diff
@@ -1,86 +1,182 @@
+	{ text: 'ガイド', header: true, type: 'learn', key: 'features' },
+	{ text: 'ルーティング', slug: 'guides/routing', key: 'guides/routing' },
+	{ text: 'Markdown', slug: 'guides/markdown-content', key: 'guides/markdown-content' },
+	{
+		text: 'スクリプトとイベントハンドリング',
+		slug: 'guides/client-side-scripts',
+		key: 'guides/client-side-scripts',
+	},
+	{ text: 'CSSとスタイル', slug: 'guides/styling', key: 'guides/styling' },
+	{ text: '画像', slug: 'guides/images', key: 'guides/images' },
+	{ text: 'フォント', slug: 'guides/fonts', key: 'guides/fonts' },
+	{ text: 'インポート', slug: 'guides/imports', key: 'guides/imports' },
+	{ text: 'エンドポイント', slug: 'guides/endpoints', key: 'guides/endpoints' },
+	{ text: 'データフェッチ', slug: 'guides/data-fetching', key: 'guides/data-fetching' },
+	{ text: '国際化', slug: 'guides/internationalization', key: 'guides/internationalization' },
+	{ text: 'ミドルウェア', slug: 'guides/middleware', key: 'guides/middleware' },
+	{ text: 'テスト', slug: 'guides/testing', key: 'guides/testing' },
+	{ text: '認証', slug: 'guides/authentication', key: 'guides/authentication' },
+	{ text: 'トラブルシューティング', slug: 'guides/troubleshooting', key: 'guides/troubleshooting' },
 
-	// Configuration
-	configuration: '設定',
-	'guides/configuring-astro': 'Astroの設定ファイル',
-	'guides/typescript': 'TypeScript',
-	'guides/aliases': 'importエイリアス',
-	'guides/environment-variables': '環境変数',
+	{ text: '設定', header: true, type: 'learn', key: 'configuration' },
+	{
+		text: 'Astroの設定ファイル',
+		slug: 'guides/configuring-astro',
+		key: 'guides/configuring-astro',
+	},
+	{ text: 'TypeScript', slug: 'guides/typescript', key: 'guides/typescript' },
+	{ text: 'importエイリアス', slug: 'guides/aliases', key: 'guides/aliases' },
+	{ text: '環境変数', slug: 'guides/environment-variables', key: 'guides/environment-variables' },
 
-	// Reference
-	reference: 'リファレンス',
-	'reference/configuration-reference': '設定方法',
-	'reference/api-reference': 'ランタイムAPI',
-	'reference/integrations-reference': 'インテグレーションAPI',
-	'reference/adapter-reference': 'アダプターAPI',
-	'reference/image-service-reference': '画像サービスAPI',
-	'reference/dev-toolbar-app-reference': 'Dev Toolbar App API',
-	'reference/directives-reference': 'テンプレートディレクティブ',
-	'reference/cli-reference': 'Astro CLI',
-	'reference/error-reference': 'エラーリファレンス',
-	'guides/publish-to-npm': 'NPMパッケージの形式',
-});
+	{ text: 'リファレンス', header: true, type: 'api', key: 'reference' },
+	{
+		text: '設定方法',
+		slug: 'reference/configuration-reference',
+		key: 'reference/configuration-reference',
+	},
+	{ text: 'ランタイムAPI', slug: 'reference/api-reference', key: 'reference/api-reference' },
+	{
+		text: 'インテグレーションAPI',
+		slug: 'reference/integrations-reference',
+		key: 'reference/integrations-reference',
+	},
+	{
+		text: 'アダプターAPI',
+		slug: 'reference/adapter-reference',
+		key: 'reference/adapter-reference',
+	},
+	{
+		text: '画像サービスAPI',
+		slug: 'reference/image-service-reference',
+		key: 'reference/image-service-reference',
+	},
+	{
+		text: 'Dev Toolbar App API',
```

Comment:
ここなのですが、この前訳していただいた

https://github.com/withastro/docs/blob/b0112d48cacf935d5eaafd33bdf14fa08a465f39/src/content/docs/ja/reference/configuration-reference.mdx#L636

の箇所で Dev Toolbar を「開発ツールバー」としているので、それに合わせて訳してしまっていいかなと思いました:

```suggestion
		text: '開発ツールバーアプリAPI',
```

Reply:
@morinokami たしかにそうですね。アップデートしました！